### ISSUE TITLE
cmakeFlags not copied to run-folder during `make install`

### ISSUE THREAD

**Issue by n01r** [ComputationalRadiationPhysics/picongpu#1098]
When compiling a parameter set and executing `make install` the file `cmakeFlags` is not copied to the run-directory. This leads to problems when trying to compile the same parameter set but from the run-directory, for example, because one notices that the code was run with a faulty parameter.

**Comment by ax3l** [comment]
as posted (and please keep the discussion there) in #1097, this makes no sense :)

`cmakeFalgs` are only needed as input for cmake. this is *before* `make` and *before* `make install`.

**Comment by ax3l** [comment]
make install does not copy to the `run` directory but to the parameter set. you problem is, that the `submitAction.sh` called by `tbg` does not copy the `cmakeFlags`.

**Comment by n01r** [comment]
Look again:
> ~/src/picongpu/configure -t 4 ~/WORKDIR/master-thesis/055_TC02_Neon_ADK_Emax\=5AU_steps\=10/picongpu/ && makePIC -j install | more

I explicitly configured an example that has already run - and I also add the flag `-t`to `configure`.
But in this folder there is ....

> submitAction.sh called by tbg does not copy the cmakeFlags

exactly what I was saying and therefore the new issue ;)

**Comment by ax3l** [comment]
sorry, but this is not the case. `make install` is not the same as a `tbg` call.

**Comment by n01r** [comment]
@psychocoderHPC and I just saw that it came from somewhere there - sorry that I have not the expertise, yet, to word it correctly right away

**Comment by n01r** [comment]
but yes, true - it comes from the `tbg` call because this is where the run-directory is specified

**Comment by ax3l** [comment]
please try to [answer](https://github.com/ComputationalRadiationPhysics/picongpu/issues/1097#issuecomment-140035555) the [questions](https://github.com/ComputationalRadiationPhysics/picongpu/issues/1097#issuecomment-140043626) you get in issues, skipping those does not evolve the issue.

let me please put this in here again: please do not use `cmakeFlags` for your cases, they are *used for integrated testing*.

Also, [if I may repeat](https://github.com/ComputationalRadiationPhysics/picongpu/issues/1098#issuecomment-140044858):

> If you want a specific test from the cmakeFlags, create a parameter set from it and remove all lines
> in cmakeFlags but the one (then moved in 0) that you need.

**Comment by ax3l** [comment]
Does this solve your problem? :)

**Comment by n01r** [comment]
Sure, I know how to resolve it now.  
Do it the same way as before and not take a shortcut.   
It only came up because I noticed that I made an error and wanted to fix it. I only keep my many different runs with different settings in the directory where all the runs are. The parameter sets to create them, though, I change frequently because they only mark the general test cases, not the NxN implementations.
